modules/nf-core/nacho/normalise/tests/main.nf.test

```diff
@@ -0,0 +1,43 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test nacho/normalise
+nextflow_process {
+
+    name "Test Process NACHO_NORMALISE"
+    script "../main.nf"
+    process "NACHO_NORMALISE"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "nacho"
+    tag "nacho/normalise"
+
+    test("Should run without failures") {
```

Comment:
Test names should describe the test dataset and configuration used:
https://nf-co.re/docs/guidelines/components/modules#test-names